config/config.go

```diff
@@ -74,13 +75,23 @@ type Config struct {
 	// libraries in the workspace for dependency resolution
 	IndexLibraries bool
 
+	// KindMap maps from a kind name to its replacement. It provides a way for
+	// users to customize the kind of rules created by Gazelle, via
+	// # gazelle:map_kind.
+	KindMap map[string]ReplacementKind
```

Comment:
(No change needed, just a comment):

I was thinking about whether it makes sense for this to be here or in `updateConfig`. I think it makes sense here. We may want to make `update-repos` observe the directive within `WORKSPACE`. That can be a TODO though.

Reply:
Yeah, the decision didn't seem super obvious to me either. SGTM.

Do you want me to add the TODO here in the code, or did you just mean as a mental note? (I usually don't add TODOs unless there are near-term plans to implement, since they tend to build up.)